### PR TITLE
docs(throttling): replace comcast with throttle and add more windows options

### PR DESCRIPTION
**Pull request by radum**
This updates the throttling docs to add a more reliable option instead of comcast -- throttle: https://www.sitespeed.io/documentation/throttle/

Also the 2 existing Windows options don't work for Windows 10 (at least in my tests) so I have added 2 more options for Windows 10 that are viable solutions, even if one is a Paid options.

### CODE REVIEW

**Comment by paulirish** [comment]
@radum dayum. just 45 minutes later. that was quick. :D

File: docs/throttling.md [hunk 1]
```diff
@@ -36,35 +36,40 @@ We plan to improve the experience of viewing a trace under simulated throttling.
...
-### Using Lighthouse with `comcast`
+### Using Lighthouse with `throttle`
 
 ```sh
 # Enable system traffic throttling
-comcast --latency=150 --target-bw=1638
+throttle 3gfast
 
 # Run Lighthouse with its own throttling disabled
 lighthouse --throttling.requestLatencyMs=0 --throttling.downloadThroughputKbps=0 --throttling.uploadThroughputKbps=0 # ...
```
Comment (paulirish) [inline docs/throttling.md:71]: @patrickhulce shouldn't this be ` --throttling-method=provided` ?

https://github.com/GoogleChrome/lighthouse/commit/70f9ecc497274e2d8e194030eb396a40f86b118f#diff-6649e57a9a2d51469c63884e7184e221 is where it was last set

Comment (radum) [reply docs/throttling.md:71]: Hmm good point, the docs are saying:

```
lighthouse <url> --emulated-form-factor=none                              Disable device emulation and all throttling.
    --throttling-method=provided
```

But not sure if they need to be used both or `--throttling-method=provided` is a shorthand of all `--throttling.` options.

Comment (patrickhulce) [reply docs/throttling.md:71]: Ah, yes it should. `--throttling-method=provided` basically tells Lighthouse to ignore whatever is set in `--throttling.*`

Comment (radum) [reply docs/throttling.md]: OK I have updated the file with `--throttling-method=provided`.

**Comment by radum** [comment]
Current failing tests look they are unrelated so I guess this could be moved as is.

**Comment by radum** [comment]
Hello,

@connorjclark hope you are well. Is there a reason this PR hasn't been merged? Was hoping to reference the new docs to some peers.

**Comment by connorjclark** [comment]
Can you merge the latest master branch to yours? Once the tests pass the PR will automatically merge.

**Pull request merged**